src/entity/create_stmt.rs

```diff
@@ -0,0 +1,172 @@
+                    None => { sea_query_column_def.double(); },
+                },
+                Sea_Query_Column_Type::Decimal(_) =>  { sea_query_column_def.decimal(); },
+                Sea_Query_Column_Type::DateTime(precision) => match precision {
+                    Some(precision) => { sea_query_column_def.date_time_len(precision); },
+                    None => { sea_query_column_def.date_time(); },
+                },
+                Sea_Query_Column_Type::Timestamp(precision) => match precision {
+                    Some(precision) => { sea_query_column_def.timestamp_len(precision); },
+                    None => { sea_query_column_def.timestamp(); },
+                },
+                Sea_Query_Column_Type::Time(precision) => match precision {
+                    Some(precision) => { sea_query_column_def.time_len(precision); },
+                    None => { sea_query_column_def.time(); },
+                },
+                Sea_Query_Column_Type::Date =>  { sea_query_column_def.date(); },
+                Sea_Query_Column_Type::Binary(length) => match length {
+                    Some(length) => { sea_query_column_def.binary_len(length); },
+                    None => { sea_query_column_def.binary(); },
+                },
+                Sea_Query_Column_Type::Boolean =>  { sea_query_column_def.boolean(); },
+                Sea_Query_Column_Type::Money(_) =>  { sea_query_column_def.money(); },
+                Sea_Query_Column_Type::Json =>  { sea_query_column_def.json(); },
+                Sea_Query_Column_Type::JsonBinary =>  { sea_query_column_def.json_binary(); },
+                Sea_Query_Column_Type::Custom(iden) => { sea_query_column_def.custom(Alias::new(&iden.to_string())); },
+                Sea_Query_Column_Type::Uuid =>  { sea_query_column_def.uuid(); },
+                Sea_Query_Column_Type::TimestampWithTimeZone(length) => match length {
+                    Some(length) => { sea_query_column_def.timestamp_with_time_zone_len(length); },
+                    None => { sea_query_column_def.timestamp_with_time_zone(); },
+                },
+            }
+            stmt.col(&mut sea_query_column_def);
+        }
+        stmt.if_not_exists();
+
+        stmt
+    }
+}
+
+impl<EntityTrait> CreateStatementOf for EntityTrait {}
+
+#[cfg(test)]
+mod tests {
+
+    use crate::{CreateStatementOf, tests_cfg};
+
+    #[test]
+    fn test_create_statement_tests_cfg_cake() {
+        let create_statement = tests_cfg::cake::Entity::create_table_statement_of(tests_cfg::cake::Entity);
+        let table = format!("{:?}", create_statement.get_table_name());
+        let columns = format!("{:?}", create_statement.get_columns());
+        let relations = format!("{:?}", create_statement.get_foreign_key_create_stmts());
+        let indexs = format!("{:?}", create_statement.get_indexes());
+        let result = format!("{:?}", create_statement);
```

Comment:
We should not be using the debug formatter to construct our test case.
@billy1624 please help changing to concrete structs

Reply:
Done!